tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -8378,6 +8378,8 @@ void ConvertRangeHalfToFloat(float *dst, DirectX::PackedVector::HALF *src,
   }
 }
 
+#if 0
```

Comment:
Could we do this as `#ifndef NDEBUG` instead?

Also since this is a template, we should add some extern template declarations to force instantiation so that we can call it inside the debugger.